site/community/release_guide.md

```diff
@@ -378,10 +381,10 @@ Start the review-and-vote thread on the dev@ mailing list. Here’s an email tem
     * All artifacts to be deployed to the Maven Central Repository [3]
     * Source code tag "release-4.5.0" [4]
 
-    BookKeeper's KEY file contains PGP keys we use to sign this release:
+    BookKeeper's KEYS file contains PGP keys we used to sign this release:
```

Comment:
why 'used'?

Reply:
Just because while sending the email you have already signed the artifacts, it is on the past, maybe my English is not very good.
Maybe I can reword as
"BookKeeper's KEYS file contains PGP keys we use to sign this releases:"